.github/workflows/maven-publish.yml

```diff
@@ -11,14 +11,26 @@ on:
         required: false
         default: "1145.vb_cf6cf6ed960"
         type: string
-  push:
-    branches: [master]
-  pull_request:
+      createRelease:
+        description: "Wether or not the build should trigger a new release on Github"
```

Comment:
Based on how this shows up we should just call it "Create Github Release" 
![image](https://user-images.githubusercontent.com/1075793/163370905-5fc4f03b-6b24-43c2-836b-9ac5bcebf622.png)